setup.py

```diff
@@ -223,8 +223,8 @@ def run(self):
     },
     py_modules=python_modules_list,
     install_requires=[
-        'onnx>=1.2.3',
-        'numpy>=1.18.0'
+        'onnx>=1.6.0,<1.7.0',
```

Comment:
ORT only works 1.6.x now? Any reason for that?
We have a bunch of nightly builds actually testing ORT and converter from onnx 1.2.3, which runs very well. 
If this PR takes effective, these nightly build test has to be stopped.

Reply:
To relax the constraint, we need to actually test it. Currently in all of our builds we only tests it with the latest onnx release.